backend/src/auth/strategies/jwt.strategy.ts

```diff
@@ -0,0 +1,24 @@
+import { ExtractJwt, Strategy } from "passport-jwt";
+import { PassportStrategy } from "@nestjs/passport";
+import { Injectable } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+
+type IJwtPayload = {
```

Comment:
Use interface or remove the "I" prefix.

Reply:
That's convention is good, but some people use "TName" or something like that, what convension did u use for types defined with 'type'?